### DISCUSSION TITLE
How can I use the same token for AndroidProvider.debug and AppleProvider.debug using the firebase_app_check plugin?

### DISCUSSION THREAD

**Discussion by adonisRodxander** [firebase/flutterfire#13480] [Q&A]
I have noticed that the debug token changes every time I uninstall and install the app. Therefore, we have to keep updating our debug tokens in the Firebase console. We are a team, so each of us has to keep our tokens updated. I was wondering if we could share the same token internally, similar to the web token that we can set in the index.html. I'm looking for the same API but for Android and iOS

**Comment by rashidotm** [comment]
I suggest adjusting your workflow to avoid uninstalling the app between builds. If you simply update or replace the app the debug token remains the same. 

Uninstalling then reinstalling the app is what triggers the creation of a new token per device.
